opa/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+{
+    'name': 'Open Academy',
+    'summary': 'Manage your academy.',
+    'author': 'Vauxoo',
+    'website': 'https://vauxoo.com',
+    'category': 'Services',
+    'version': '15.0.0.1.0',
```

Comment:
First version should be:

```suggestion
    'version': '15.0.1.0.0',
```

Reply:
Thanks, I will change it. Just out of curiosity, should versions follow a certain versioning scheme?